src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/CoreWindowExtension.cs

```diff
@@ -43,13 +42,13 @@ public CoreWindowExtension(object owner)
 			{
 				_libInputContext = libinput_path_create_context();
 
-				_inputThread = new Thread(Run)
+				var inputThread = new Thread(Run)
```

Comment:
This would make inputThread collectible, I'm not too sure if this is the appropriate way to handle this.

Reply:
I can just suppress the warning for this case.